### DISCUSSION TITLE
My windows still Bricked! You should take down this script!

### DISCUSSION THREAD

**Discussion by annarana** [ionuttbara/windows-defender-remover#155] [General]
It's been two months since the last update. Did you gave up on this script? Are you gonna let people keep getting their computers bricked? I'm sorry but you really should warn people that this is unstable before they install your program. My taskbar is non existent, I keep getting black screens sometimes, all the apps from the windows store doesn't work, and now even the four finger gestures on my touchpad doesn't work so If I minimize chrome is lost forever. I tried to reinstall windows 11 using installation media to not lose all the programs, but I'm still getting the same problems. Yes, I know, I should do a clean install and install all my programs again, but it really piss me off to do this as I have loads of programs to reinstall and don't have a drive to do a backup right now. At least say something, I've been writing my issues for the past month but you haven't replied a single message.  Thanks

**Comment by BeegDeek** [comment]
It worked for me and nothing is bricked and all apps work
